deployment/refresh-views.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
Would it make sense to invert the structure of this folder a bit? 

Current:
- deployment/refresh-views.sh
- deployment/sql/materialized-views/*.sql

Proposed:
- deployment/materialized_views/refresh-views.sh
- deployment/materialized_views/sql/*.sql

This keeps folders under `deployment/` loosely organized by topic, e.g. batch, streaming, materialized_views and keeps a script (refresh-views) intended to be run on infrastructure, out of the root deployment directory which currently contains artifacts and scripts to be run locally that build said infrastructure.

EDIT: Looking at the dir, it appears we also have create table statements as well. Perhaps those could be moved to `deployment/db-init/*.sql`. Or, this is now too complicated and disregard this comment 😄

Reply:
In order to address the mess of docker build resources, I pulled `refresh-views.sh` into a new directory.  That sorta makes it unnecessary to make a `materialized-views` directory.  I think the `sql` directory name is pretty clear?  There's a README in that directory that should clear up confusion.  Happy to revisit if needed.

Reply:
Updated changes are great.